docs/en/operation/StressBench.md

```diff
@@ -111,3 +111,69 @@ This process continues until the max throughput is discovered.
 This algorithm uses many iterations and time to determine the throughput, but it produces a more consistent result, and the actual number is meaningful.
 If the resulting throughput is X, that means if the clients produce X throughput, the server can achieve that.
 It also means if the clients produce more than X throughput, like X+N, the server will not be able to achieve X+N throughput.
+
+
+## Fuse IO Stress Bench
+The Fuse IO Stress Bench is a tool to measure the reading performance of Alluxio Fuse-based POSIX API.
+
+### Procedure
+
+#### 0. Prerequisite
+A running Alluxio cluster is required. The Alluxio path for testing needs to be mounted to the local file system. To prevent Fuse cache affecting
```

Comment:
Can you reconstruct the prerequisite into multiple lines?
- A running Alluxio cluster
- Mount the testing Alluxio path to a local address (link to Alluxio POSIX API)
- (Optional) To prevent the effect of operating system caching and POSIX API caching, using different mount points for writing and reading can produce more accurate benchmarking results.